### ISSUE TITLE
Running retro-go on a ST7735 display

### ISSUE THREAD

**Issue by lordherp** [ducalex/retro-go#36]
Love this project but I see it only supports ILI9341 based displays. I have a couple displays laying around which are ST7735 driver based (Like 1.8" inch TFT LCD Display module ST7735S 128x160 for example). Is there anyway I could run this on ST7735 lcd?

**Comment by ducalex** [comment]
Yeah these controllers are largely identical so it should work, but you'll have to add the init code specific to your display in `rg_display.c -> lcd_init()`.

However keep in mind that retro-go doesn't support downscaling, only upscaling, so the image will be cropped in most emulators.
